### ISSUE TITLE
Homepage. <br> causes pause and repetition in sentence when using voiceover

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#107]
### Description of issue

Using voiceover on Chrome, the counter reads as follow:
1. 164
2. Services utilisant
3. Services utilisant
4. GC Notification

When navigating backwards:
1. GC notifications
2. GC notification
3. Services utilisant
4. 164

<img width="1176" alt="Capture d’écran, le 2021-10-14 à 12 08 36" src="https://user-images.githubusercontent.com/5230720/137355685-4a6fd2ba-28af-419b-835e-a5b3379276f4.png">

I suspect the `<br>` is the cause of this weird behaviour. Removing it, and adding a non-breaking space between "GC" and "Notification" seems to be fixing this. 

The non breaking space is not part of the solution, but will prevent the two parts of the name breaking across two lines. 

### Potential fix

- remove the `<br>`
- add non breaking space between "GC" and "Notification"
- Bonus point for not breaking the number and the description. This should be read as a sentence and would benefit from being contained in a single `<p>` with a span to style either the number or the sentence. See comment below
